keel-clouddriver/src/test/kotlin/com/netflix/spinnaker/keel/clouddriver/MemoryCloudDriverCacheTest.kt

```diff
@@ -48,20 +44,21 @@ object MemoryCloudDriverCacheTest {
   )
 
   @Test
-  fun `security groups are looked up from CloudDriver`() {
-    whenever(cloudDriver.getSecurityGroups("prod")) doReturn securityGroups
+  fun `security group summaries are looked up from CloudDriver`() {
```

Comment:
these are somewhat lame tests against mocks but they were pre-existing so I've updated for now